onnxruntime/core/providers/webnn/builders/impl/split_op_builder.cc

```diff
@@ -179,6 +142,23 @@ bool SplitOpBuilder::IsOpSupportedImpl(const InitializedTensorSet& initializers,
       }
     }
   }
+
+  if (split.size()) {
+    int64_t sum = 0;
+    // TODO: Allow 0 size dimensions.
+    // https://github.com/webmachinelearning/webnn/issues/391
+    for (size_t i = 0; i < split.size(); i++) {
```

Comment:
```suggestion
    for (uint32_t split_value : splits) {
```
Unless you need the same index for multiple arrays, it's more robust to use a ranged `for` loop (fewer chances of typos when other people refactor your code later).